ui/lib/components/menu/menu.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:html';
+
+import 'package:katikati_ui_lib/components/tooltip/tooltip.dart';
+
+class MenuActionItem {
+  DivElement renderElement;
+  Function onClick;
+
+  MenuActionItem(this.renderElement, this.onClick) {
+    renderElement.onClick.listen((e) {
+      e.stopPropagation();
+      onClick();
+    });
+  }
+}
+
+/*
+<span class="menu-wrapper">
+  <span class="menu-icon">
+    <icon />
+  </span>
+  <div class="menu-items-wrapper">
+    <div class="menu-item">Item 1</div>
+    <div class="menu-item">Item 2</div>
+  </div>
+</span>
+*/
```

Comment:
Can we BEM-ify the CSS classes?

```suggestion
<span class="menu">
  <span class="menu__icon">
    <icon />
  </span>
  <div class="menu__items">
    <div class="menu__item">Item 1</div>
    <div class="menu__item">Item 2</div>
  </div>
</span>
*/
```